src/LibChorus/VcsDrivers/Mercurial/HgRepository.cs

```diff
@@ -1475,10 +1471,28 @@ private static void SetExtensions(IniDocument doc, IEnumerable<KeyValuePair<stri
 			}
 		}
 
-		internal static bool CheckExtensions(IniDocument doc, IEnumerable<KeyValuePair<string, string>> extensionDeclarations)
+		/// <summary/>
+		/// <returns>
+		/// Returns true if the extensions in the repo contain only the extensions declared in the given map.
+		///</returns>
+		internal static bool CheckExtensions(IniDocument doc, Dictionary<string, string> extensionDeclarations)
```

Comment:
Right, but Jason is going to some trouble to make sure there are no extensions except the ones we want. I don't know why, but if it's crucial that something NOT be active, we might need to worry about the global settings activating it.